src/sinks/kafka/service.rs

```diff
@@ -51,6 +47,10 @@ impl DriverResponse for KafkaResponse {
             output: None,
         }
     }
+
+    fn bytes_sent(&self) -> Option<(usize, &str)> {
+        Some((self.bytes_sent, "tcp"))
```

Comment:
No longer `kafka`?

Reply:
I changed it intentionally because I saw kafka is operating over tcp. So this seemed more correct to me? But I can change it back if `kafka` is more appropriate.

Reply:
`Kafka uses a binary protocol over TCP.`

I feel like `kafka` was intentional originally, but we could double check it. I'd consider this a _sorta breaking change_.

Reply:
After your comment about the registered event, and Bruce's response, I'm going to back this out to avoid regression.